tableschema/types/array.py

```diff
@@ -4,42 +4,21 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import six
 import json
-from .. import exceptions
-from .. import helpers
-from . import base
+from ..config import ERROR
 
 
 # Module API
 
-class ArrayType(base.JTSType):
-
-    # Public
-
-    name = 'array'
-    null_values = helpers.NULL_VALUES
-    supported_constraints = [
-        'required',
-        'pattern',
-        'enum',
-        'minLength',
-        'maxLength',
-    ]
-    # ---
-    python_type = list
-
-    def cast_default(self, value, fmt=None):
-
-        if isinstance(value, self.python_type):
-            return value
-
+def cast_array(format, value):
```

Comment:
@roll I'm not convinced it was a good idea to make casting internal. simple casting of values is really useful. Do you remember I wanted us to use https://github.com/pudo/typecast directly for this back then?

Whatever the case, I agree with @amercader that, as format is essentially optional (defaults to `default`, we don't require users to provide `default`), then a more conventional function signature would be with format as a keyword argument.